packages/ketcher-react/src/script/editor/utils/customOnChangeHandler.ts

```diff
@@ -40,7 +40,9 @@ type Data = {
 
 export function customOnChangeHandler(action, handler) {
   const data: Data[] = [];
-
+  if (!action) {
```

Comment:
Let's prepare this function a bit for some future adjustments. Let it be wrapper for both micro and macro 'changes' event handler and we will have two separate functions for micro and macro event handlers. The former will have all the existing functionality – preparing the data based on passed action and invoking the handler with this data. The latter will just have invoking the handler with empty arguments list – just what you've added here

Then, based on the currently active editor (you can use `window.isPolymerEditorTurnedOn`) just invoke either the first or the second function.